tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1289,13 +1235,80 @@ class CompletionSuite extends BaseCompletionSuite {
        |""".stripMargin
   )
 
-  checkItems(
-    "scope-completions-empty-query",
```

Comment:
We remove this test?

Reply:
I've modified in locally in attempt to fix #3140 but it wasn't a case for quick fix.
Then I realized that this test isn't really valuable as it doesn't check result.

I might return as it is. Or it will be added later with a fix for #3140 and more strict result validation